src/test/resources/localstack/setup-es.sh

```diff
@@ -6,6 +6,12 @@ export AWS_SECRET_ACCESS_KEY=foobar
 export AWS_DEFAULT_REGION=eu-west-2
 
 aws --endpoint-url=http://localhost:4566 es create-elasticsearch-domain --domain-name es1 > /dev/null
-mkdir /tmp/localstack/es_backup || echo "Failed to create /tmp/localstack/es_backup folder"
+if [[ -d /tmp/localstack/es_backup ]]
+    then
```

Comment:
forced double indent - we normally do `; then` on the same line as the `if`

Reply:
[_See latest commit_](https://github.com/ministryofjustice/prisoner-offender-search/pull/166/commits/1f81f57f58238a61d9ef1e17e28b6cb8afcc0f33).